tasks/rh-sign-image-cosign/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+#
+# Install the CRDs so we can create/get them
+.github/scripts/install_crds.sh
+
+# Add RBAC so that the SA executing the tests can retrieve CRs
+kubectl apply -f .github/resources/crd_rbac.yaml
+
+# Create a dummy cosign-secret secret (and delete it first if it exists)
```

Comment:
As I commented [here](https://github.com/konflux-ci/release-service-catalog/pull/468/files#r1669971195) I'd suggest to call this secret `cosignSecretName`.